ROADMAP.md

```diff
@@ -1,6 +1,8 @@
 # Roadmap
 
-The roadmap for the  next release (v0.11, expected February 2024) is the following:
+The next release of Liqo will include many breaking changes, due to the necessity to bring more modularity to the project, a cleaner and more granular control (e.g., public APIs to control the most important features of Liqo), and other features.
+Hence, the Liqo team has decided to release the most breaking updates within a single release, hence limiting the disruption (and potentially painful migration strategies) to the current adopters.
+Hence, the original plans to release v0.11 in Feb 2024 have been rescheduled to Jun 2024, with the following main features:
 
 * [Feature] Complete re-design of the network fabric, involving a new communication model _inside_ the cluster (i.e., pod-based communication instead of host-based services; geneve instead of vxlan) and a new, simplified model for the intra-cluster connectivity (still based on Wireguard, but more robust and open to other technologies).
 * [Feature] CRD-based network modularity. All the network components can be individually controlled with new CRDs, hence enabling the different components of Liqo to be turned on/off and configured in a more granular way.
```

Comment:
```suggestion
* [Feature] Network module: complete re-design of the network fabric, involving a new communication model _inside_ the cluster (i.e., pod-based communication instead of host-based services; `geneve` instead of `vxlan`) and a new, simplified model for the intra-cluster connectivity (still based on Wireguard, but more robust and open to other technologies).
All the network components can be individually controlled with new CRDs, hence enabling the different components of Liqo to be turned on/off and configured in a more granular way.
* [Feature] Authentication module: the peering authentication between clusters has been greatly simplified and is now fully declarative.
It does not require exposing a dedicated auth service like in old versions and no API server exposition is necessary from the consumer side.
The new `ResourceSlice` CR allows a more granular and flexible control of the resources requested to cluster providers.
* [Feature] Offloading module: it is now possible to have multiple virtual nodes targeting the same remote provider cluster.
This allows, for example, to split the resources of bigger clusters across multiple virtual nodes, or to have nodes with specific resources (e.g. GPUs) or characteristics (e.g. specific architectures).
It can also be used to share huge resource pools with another cluster while keeping the virtualnodes size quite small, avoiding a "black hole" effect during scheduling.

There are many more news and features, and we are planning to publish some blog posts to address those.

```

Reply:
I cannot fully understand what "pod-based communication instead of host-based services" means. Is it "pod-based communication between user pods and the Liqo gateway instead of host-based services"?